kodilla-collections/src/main/java/com/kodilla/collections/sets/homework/Stamp.java

```diff
@@ -0,0 +1,51 @@
+package com.kodilla.collections.sets.homework;
+
+import java.util.Objects;
+
+public class Stamp {
+    private String stampName;
+    private String sizeStamp;
```

Comment:
w zadaniu bylo wymiary, wiec na moje oko powinny byc dwa pola typu double :) Ale to juz maly szczegol jest, wiec nie musisz twego zmieniac juz.